client/me/purchases/manage-purchase/change-payment-method/tos-text.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import { localizeUrl } from 'calypso/lib/i18n-utils';
+import { AUTO_RENEWAL, MANAGE_PURCHASES } from 'calypso/lib/url/support';
+
+export default function TosText(): JSX.Element {
+	const translate = useTranslate();
+	return (
+		<>
+			{ translate(
+				'You agree to our {{tosLink}}Terms of Service{{/tosLink}}, and if you use it to pay for a subscription or plan, you authorize your payment method to be charged on a recurring basis until you cancel, which you can do at any time. You understand {{autoRenewalSupportPage}}how your subscription works{{/autoRenewalSupportPage}} and {{faqCancellingSupportPage}}how to cancel{{/faqCancellingSupportPage}}.',
```

Comment:
I think we should add "By saving a payment method," at the beginning of the first sentence (to match the "By saving a credit card," that was there before).  Otherwise the first "it" the sentence is pretty ambiguous.

Reply:
Good point. I mashed together the previous TOS here with the TOS from checkout and I didn't notice that wording issue.